src/main/java/com/github/dockerjava/api/command/ListServicesCmd.java

```diff
@@ -18,17 +18,36 @@
     Map<String, List<String>> getFilters();
 
     /**
-     * @param ids
-     *            - Show only services with the given ids
+     * @param ids - Show only services with the given ids
      */
     ListServicesCmd withIdFilter(List<String> ids);
+
+    /**
+     * @param names - Show only services with the given names
+     */
+
+    ListServicesCmd withNameFilter(String... names);
```

Comment:
method with list already exist, why do you nead additional?

Reply:
existing signature is List, add Varargs to get consistent with existing ListTasksCmd

Reply:
the idea is to stop using non collection types

Reply:
IMHO, sometimes varargs is more convenient, e.g. `withIdFilter(id)` is convenient  than cast to List `withIdFilter(Collections.singletonList(id))`, anyway, I removed the varargs